Documentation/network/egress-gateway/egress-gateway.rst

```diff
@@ -311,7 +311,9 @@ the specification above:
           class: mediabot
           # The following label selects default namespace
           io.kubernetes.pod.namespace: default
-
+      nodeSelector: # optional, if not specified the policy applies to all nodes
+        matchLabels:
+          node.kubernetes.io/name: node1 # only traffic from this node will be SNATed
```

Comment:
Is it possible for a `nodeSelector` to be used without a `podSelector`? Or does a `podSelector` always need to be specified?